docs/en/latest/admin-api.md

```diff
@@ -657,12 +659,44 @@ After the execution is successful, nodes will not retain the original data, and
 
 ```
 
+Example 2: How to proxy upstream https.
```

Comment:
Bad title, I think "How to proxy requests to upstream with HTTPS scheme" is better.